vnext/Microsoft.ReactNative/RedBox.cpp

```diff
@@ -309,13 +309,14 @@ struct RedBox : public std::enable_shared_from_this<RedBox> {
     m_stackPanel.Children().Clear();
     for (const auto frame : m_errorInfo.Callstack) {
       const winrt::hstring xamlFrameString =
-          L"<StackPanel Margin='0,5,0,5'"
-          L"  xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'"
-          L"  xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'"
-          L"  >"
-          L"  <TextBlock x:Name='MethodText' FontSize='14' Foreground='White' TextWrapping='Wrap' FontFamily='Consolas'/>"
-          L"  <TextBlock x:Name='FrameText' FontSize='14' Foreground='Gray' TextWrapping='Wrap' FontFamily='Consolas'/>"
-          L"</StackPanel>";
+          LR"(
```

Comment:
>LR"( [](start = 10, length = 4)

Consider changing it to LR"XAML( ... )XAML" to increase readability and uniqueness of the delimiters.